embedded-hal-nb/src/serial.rs

```diff
@@ -33,3 +35,21 @@ impl<T: Write<Word>, Word: Copy> Write<Word> for &mut T {
         T::flush(self)
     }
 }
+
+/// Implementation of `core::fmt::Write` for the HAL's `serial::Write`.
+///
+/// TODO write example of usage
```

Comment:
TODO: example

Reply:
The TODO was already there :sweat_smile: